### PR TITLE
Add docs for continuous logging mode

### PR DESCRIPTION
**Pull request by sjp4**

### CODE REVIEW

File: docs/android/android-logging.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+:::info
+
+Logs can also be collected via bug reports - but this is not recommended.
+
+:::
+
+## Configuration
+
+Configure logging on the
+[settings dashboard](https://app.memfault.com/organizations/-/projects/-/settings/data-sources).
```
Comment (tyhoff) [inline docs/android/android-logging.mdx:23]: Would be more explicit about what settings. 

```suggestion
[Project Settings page under **Data Sources**](https://app.memfault.com/organizations/-/projects/-/settings/data-sources).
```

File: docs/android/android-logging.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+![](/img/docs/android/android-logging-config.png)
+
+### Kernel Oopses
+
+Kernel Oopses are extracted from log files, to create traces.
```
Comment (tyhoff) [inline docs/android/android-logging.mdx:29]:
```suggestion
Kernel Oopses are extracted from log files to create traces.
```

File: docs/android/android-logging.mdx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+has particularly spammy logging), it is possible that some logs may be missed.
+
+### Continuous
+
+:::info
+
+Continuous logging mode is a preview feature, available from Bort 4.3.0 onward.
+
+Any devices running an older version of Bort will continue to use Periodic mode
+if **Continuous** is enabled.
```
Comment (tyhoff) [inline docs/android/android-logging.mdx:48]: Did you mean "unless Continuous mode is enabled"?

**Review by tyhoff — APPROVED** [review]

File: docs/android/android-logging.mdx (hunk 3 above)
Comment (sjp4) [reply docs/android/android-logging.mdx:48]: No - I'm saying that even if you enable continuous in settings, an old sdk version will still use periodic.

**Pull request merged**